### PR TITLE
Added isClean method and corresponding test

### PR DESCRIPTION
**Pull request by asgaines**
### What is the problem / feature ?
- There was no easy way to tell if there are no dirty properties in the entity
### How did it get fixed / implemented ?
- Added a method which allows for this functionality
- Also added test (thanks go to @Danno040)
### How can someone test / see it ?
- run phpunit test, ensure all pass

### CODE REVIEW

**Comment by smazurov** [comment]
I would love to know why travis didn't test this.

**Comment by smazurov** [comment]
This LGTM, @Danno040?

File: src/Entity.php [hunk 1]
```diff
@@ -243,4 +243,12 @@ protected function normalizeAccessor($key, $set = true)
         $prefix = ($set) ? 'set' : 'get';
         return $prefix . $methodName;
     }
+
+    /**
+     * @return bool
+     */
+    public function isClean()
```
Comment (smazurov) [inline src/Entity.php]: positioning wise, it goes Pub->Prot->Private in the class.

**Comment by Danno040** [comment]
Yeah. :shipit: once the method is in the right place.

**Comment by asgaines** [comment]
:+1:

**Pull request closed without merging**